lib/hashes.md

```diff
@@ -5,40 +5,53 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A dictionary like collection of data that pairs unique keys to their corresponding vaules.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+Keys can be any data type.
+Strings and numbers would be the two most common data types. 
+
```

Comment:
Right. Could you please say more here? What are some examples of when you might choose to use a Hash to store information?